src/components/WwwFrame/LendMenu/TheLendMenu.vue

```diff
@@ -149,6 +156,12 @@ export default {
 			});
 			this.apollo.watchQuery({ query: publicLendMenuQuery }).subscribe({
 				next: ({ data }) => {
+					const version = _get(data, 'experiment.version');
+					const { enabled } = getExperimentSettingCached(this.apollo, lendMenuExpKey);
+					if (enabled) {
+						trackExperimentVersion(this.apollo, this.$kvTrackEvent, 'TopNav', lendMenuExpKey);
```

Comment:
For MARS-90 the tracking event action needs to be `EXP-MARS-89-Apr2022`. By default the `trackExperimentVersion` function uses `key` as the action, so I think what we should do is modify the `trackExperimentVersion` function to have an optional 5th argument called `action`. When `trackExperimentVersion` calls `trackEvent` it can pass in `action ?? key` for the action argument, so that it still falls back to using `key` if `action` is not defined.

Reply:
Done!